_data/accessibility-tests/tooltip.yml

```
@@ -0,0 +1,86 @@
+title: Tooltip
+component_status: pass
+test_items:
+# General tests
+  - summary: Color is not the only method used to communicate the purpose of the tool tip.
```

@sarah-sch @finekatie 
Sometimes we call this "tooltip" and other times "tool tip". Is it ok to use both?
```suggestion
  - summary: Color is not the only method used to communicate the purpose of the tooltip.
```

---

I'd suggest using the same term throughout for consistency. The only time I'd change it is when there's a concern about meaning confusion.